StudCity.Frontend/src/pages/ContactUs/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+
+
+function ContactUs() {
+    const classInput = "border-b-2 w-80 my-2 py-4 hover:border-b-zinc-400 focus:outline-none focus:border-b-black"
+    const [details, setDetails] = useState({});
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        console.log(details);
+    }
+
+  return (
+    <div className="flex w-full h-screen">
+        <div className='w-2/5 relative'>
+            <img className='h-screen z-0' src="https://images.unsplash.com/photo-1523966211575-eb4a01e7dd51?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTJ8fGNvbnRhY3QlMjB1c3xlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60" alt="" />
```

Comment:
it's probably better to put this image inside global constants file, or in this file locally